mars-sim-core/src/main/java/com/mars_sim/core/SimulationConfig.java

```diff
@@ -268,7 +270,7 @@ private String loadValue(Element parentConfig, String child) {
 		String str = childItem.getAttributeValue(VALUE);
 
 
-		if ((str == null) || str.trim().length() == 0)
+		if ((str == null) || str.trim().isEmpty())

```

Comment:
we can even use [isBlank()](https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java/lang/String.html#isBlank()) method in future in cases like this as I was told that it's a bit more efficient than .trim().isEmpty().

> if ((str == null) || str.is.blank())